pkg/helm/helm.go

```diff
@@ -83,3 +87,13 @@ func (m *Mixin) ValidatePayload(b []byte) error {
 func (m *Mixin) getKubernetesClient(kubeconfig string) (k8s.Interface, error) {
 	return m.ClientFactory.GetClient(kubeconfig)
 }
+
+func (m *Mixin) getHelmClientVersion() string {
```

Comment:
I'm wondering if we can do without the getters/setters if the struct field (`HelmClientVersion`) is exported like it is now?  Might be simpler at this point.  I'm not averse to just getting/setting via `m.HelmClientVersion` directly...

Reply:
Yes, you're right...This getter/setter certainly come from my  .NET background :)